WalletWasabi.Gui/Tabs/WalletManager/RecoverWalletView.xaml

```diff
@@ -29,6 +29,12 @@
               </DataTemplate>
             </ItemsControl.ItemTemplate>
           </ItemsControl>
+          <CheckBox IsChecked="{Binding ShowAdvancedOptions,Mode=TwoWay}"><TextBlock Text="Show advanced options"/></CheckBox>
```

Comment:
Formatting. Sorry, from next version we'll want to fix up the code smells and formatting inconsistencies.
```suggestion
          <CheckBox IsChecked="{Binding ShowAdvancedOptions, Mode=TwoWay}">
              <TextBlock Text="Show advanced options"/>
          </CheckBox>
```

Reply:
There should also be a spacing/padding or something here: 8 I think. Did you try if it looks a bit better?